test/test_arraycontext.py

```diff
@@ -778,16 +779,16 @@ def test_container_scalar_map(actx_factory):
             map_reduce_array_container, rec_map_reduce_array_container,
             )
 
-    for ary in arys:
+    for size, ary in zip(expected_sizes, arys[:-1]):
```

Comment:
The last of `arys` is just a scalar and `actx.np.array_equal` doesn't work on scalars, so this skips it now.